drivers/storage/ide/atapi/hw.c

```diff
@@ -0,0 +1,878 @@
+/*
+ * PROJECT:     ReactOS ATA Port Driver
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Request handling
+ * COPYRIGHT:   Copyright 2024 Dmitry Borisov (di.sean@protonmail.com)
+ */
+
+/* INCLUDES *******************************************************************/
+
+#include "atapi.h"
+
+/* FUNCTIONS ******************************************************************/
+
+static
+VOID
+AtaPciIdeDmaPrepare(
+    _In_ PATAPORT_CHANNEL_EXTENSION ChannelExtension)
+{
+    PPCIIDE_INTERFACE PciIdeDma = &ChannelExtension->PciIdeInterface;
+    PUCHAR IoBase = PciIdeDma->IoBase;
+
+    /* Clear the interrupt and error bits in the status register */
+    ATA_WRITE(IoBase + DMA_COMMAND, DMA_COMMAND_STOP);
+    ATA_WRITE(IoBase + DMA_STATUS, DMA_STATUS_INTERRUPT | DMA_STATUS_ERROR);
+
+    /* Beginning of the the PRD Table */
```

Comment:
```suggestion
    /* Beginning of the the Physical Region Descriptor (PRD) Table */
```
Might make sense to write it out at least once at one location where it is already used, e.g. here.